src/store/http.rs

```diff
@@ -0,0 +1,46 @@
+use std::collections::HashMap;
+
+use pyo3::{exceptions::PyValueError, pyclass, Bound, PyAny, PyErr, PyResult};
+use pyo3_stub_gen::derive::gen_stub_pyclass;
+use zarrs::storage::ReadableWritableListableStorage;
+
+use super::opendal_builder_to_sync_store;
+
+#[derive(Debug, Clone)]
+#[gen_stub_pyclass]
+#[pyclass]
+pub struct HttpStoreConfig {
+    #[pyo3(get, set)]
+    pub root: String,
```

Comment:
this should be called “endpoint”, right?

Reply:
Oops, yeah it shouldn't be root. Might need following up soon though: https://github.com/zarr-developers/zarr-python/issues/1962